ext/sqlite3/database.c

```diff
@@ -66,6 +66,23 @@ static VALUE rb_sqlite3_open_v2(VALUE self, VALUE file, VALUE mode, VALUE zvfs)
   return self;
 }
 
+static VALUE rb_sqlite3_disable_quirk_mode(VALUE self)
+{
+#if defined SQLITE_DBCONFIG_DQS_DDL
+  sqlite3RubyPtr ctx;
+  Data_Get_Struct(self, sqlite3Ruby, ctx);
+
+  if(!ctx->db) return Qfalse;
+
+  sqlite3_db_config(ctx->db, SQLITE_DBCONFIG_DQS_DDL, 0, (void*)0);
+  sqlite3_db_config(ctx->db, SQLITE_DBCONFIG_DQS_DML, 0, (void*)0);
+
+  return Qtrue;
+#else
+  return Qfalse
```

Comment:
Feels like both branches should be covered with at least building tests to avoid such stuff :(

Reply:
Avoid such what?

Also I really don't see what there is to cover, either the function exist and we use it, or either it's a noop.

Reply:
Avoid compilation error because of missing semicolon after Qfalse.

Reply:
https://github.com/sparklemotion/sqlite3-ruby/pull/324

Reply:
🤦 

Getting both versions of sqlite3 on CI is tricky, but we can probably have an ENV var flag.